Pointers/UniquePtr.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+#include <iostream>
+
+using std::cout;
```

Comment:
`using` in header files (and in inl files as well) are not recommended because everyone who includes your header will also get the using and it may create collisions with existing implementations
Also, it "flattens" the namespace but sometimes this is not a desired behaviour